### ISSUE TITLE
[tensorflow.org] Wrong footer position

### ISSUE THREAD

**Issue by rongjiecomputer** [tensorflow/tensorflow#8832]
In a few pages of tensorflow.org (like https://www.tensorflow.org/install/install_windows), the footer appears at the wrong place.

The fix is to move 
```
<footer class="devsite-utility-footer">...</footer>
<footer class="devsite-footer-linkboxes nocontent devsite-footer-linkboxes-all-backup">...</footer>
```
after
```
<div class="devsite-main-content clearfix">...</div>
```

**Comment by gunan** [comment]
@wolffg could you reassign appropriately?

**Comment by msaroufim** [comment]
@gunan Have you considered open sourcing the docs as well?

**Comment by gunan** [comment]
all the docs are already opensource.
Here is the document in question:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/docs_src/install/install_windows.md

**Comment by wolffg** [comment]
The footer location is not available to edit externally.

In this case (install_windows), there is an unclosed <table> tag which is
causing issues.  I will fix this and republish.  Thanks for the note, and
if you find other pages with weird footers, please report them.

**Comment by rongjiecomputer** [comment]
@wolffg  Ah, I see where is the trailing `<table>` now, do you want me to send a PR for this?

Also, I think `>>>` in the following snippet (same page) should be removed, the markdown rendering engine used by tensorflow.org confuses it with blockquote (Github's is fine with it though).

```md
```python
>>> import tensorflow as tf
>>> hello = tf.constant('Hello, TensorFlow!')
>>> sess = tf.Session()
>>> print(sess.run(hello))
```

**Comment by wolffg** [comment]
Don't worry about it.  I already submitted something; it should be
published soon.

**Comment by wolffg** [comment]
Closed, see #8960
